config/examples/Creality/Ender-3/MKS Robin E3/readme.md

```diff
@@ -0,0 +1,12 @@
+NOW MKS has a V1.1 Version of MKS Board 
+
+Need to compile using another board pin
+the new definition for V1.1 is
+#define MOTHERBOARD BOARD_MKS_ROBIN_E3_V1_1 instead   #define MOTHERBOARD BOARD_MKS_ROBIN_E3
```

Comment:
`BOARD_MKS_ROBIN_E3_V1_1` doesn’t exist in the main Marlin repo, so this config won’t work until support is added.

Reply:
Latest Bugfix

![Screenshot_7](https://user-images.githubusercontent.com/24917659/109395005-cf53b180-7908-11eb-85dc-750de99edbb9.png)

Reply:
Yes, I see it now. I must have been on a different branch when I looked.

These were added in https://github.com/MarlinFirmware/Marlin/pull/20216.